index.html

```diff
@@ -1850,16 +1849,52 @@ <h2>Resource Integrity</h2>
 guaranteed between implementations.
         </p>
 
-        <p class="issue atrisk" title="Multibase and Multihash are being standardized at IETF">
-The [[MULTIBASE]] and [[MULTIHASH]] specifications have been dispatched at IETF
-to be standardized in a
-<a href="https://mailarchive.ietf.org/arch/browse/multiformats/">
-Multiformats Working Group</a>. If the specifications are stabilized before this
-specification goes to the Proposed Recommendation phase, the tables above will
-be replaced with normative references to the Multibase and Multihash
-specifications.
+      </section>
+
+      <section>
+        <h2>Resource Integrity</h2>
+
+        <p class="issue atrisk" title="Unification of cryptographic hash expression formats are under discussion">
+The Working Group is currently attempting to determine whether cryptographic hash
+expression formats can be unified across all of the VCWG core specifications.
+Candidates for this mechanism include `digestSRI` and `digestMultibase`. There
```

Comment:
I don't see where in this specification the digestSRI property is defined.

Reply:
`digestSRI` is defined in VCDM -- https://w3c.github.io/vc-data-model/#integrity-of-related-resources

Reply:
> digestSRI is defined in VCDM -- https://w3c.github.io/vc-data-model/#integrity-of-related-resources

See also https://github.com/w3c/vc-data-model/pull/1296 which formalizes that.